msb/zmq_base/Payload.py

```diff
@@ -0,0 +1,57 @@
+import json
+from datetime import datetime
+
+
+class Payload:
+    def __init__(self):
+        self.data = "hello world"
+        self.name = "payload"
+        self.owner = "msb"
```

Comment:
What does the owner attribute do, or rather: what's the intention here? Is it designed as a field telling any receiving service, who send the data? won't that be covered by the topic?